### PR TITLE
Update default accuracy to None to support null JSON values

### LINKED ISSUE #674: Remove `required` attribute from unidirectional serializers
## Problem

Most serializers in the API are unidirectional, in that they serialize instances to JSON but not vice versa. These serializers have no use for the [`required` attribute](https://www.django-rest-framework.org/api-guide/fields/#required).

## Solution
The `required` argument should be removed from all output serializers.

## Additional context

To be fair, the `required` prop has a notable effect during serialization. If set to `False` fields with the value `null` are omitted entirely, which is counterintuitive and may be surprising, so we better leave it to the default value of `True`.

## Implementation

- [ ] 🙋 I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by sahil-R**
## Fixes

Fixes #674 by @dhruvkb

## Description

The issue stated that the serializers were not allowing null values to be passed on in the response  JSON which is returned by the API,The code was running fine when tested however the accuracy were not accompanied with the tags which were added to the response,added the fix with default accuracy as None and edited the test cases for the same.

## Testing Instructions

just do any API request and the output runs fine as usual with null values returned as well.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [ ] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: api/catalog/api/serializers/media_serializers.py
Comment (dhruvkb) [inline api/catalog/api/serializers/media_serializers.py:280]: Setting accuracy to a default of zero suggests that the tags are completely inaccurrate! Setting it to `None` has the intended meaning that we are uncertain about the accuracy.

```suggestion
        default=None,
```

**Review by dhruvkb — CHANGES_REQUESTED** [review]
Please remove the root `Pipfile` and `Pipfile.lock`. Both the `api/` and `ingestion_server/` have their Python environments in their directories.

File: api/catalog/api/serializers/media_serializers.py
Comment (sahil-R) [reply api/catalog/api/serializers/media_serializers.py:280]: Thank you dhruv, will do the same and revert as soon as possible

Comment (sahil-R) [reply api/catalog/api/serializers/media_serializers.py]: Done

**Review by dhruvkb — APPROVED** [review]

**Comment by dhruvkb** [comment]
This is looking good now, thanks @sahil-R!

**Comment by sahil-R** [comment]
> This is looking good now, thanks @sahil-R!


Hey @dhruvkb , Thank you for the update, can you suggest me another  approver for these changes since it requires two approvals.

**Comment by dhruvkb** [comment]
I tagged '@WordPress/openverse-api' on this issue, which picked @AetherUnbound as the second reviewer.

File: api/catalog/api/serializers/media_serializers.py
Comment (sarayourfriend) [inline api/catalog/api/serializers/media_serializers.py:281]: A bit of a nit-pick: @dhruvkb Would we need to update this documentation to say "Human-generated tags have a `null` accuracy field" rather than not having the field at all?

Comment (sahil-R) [reply api/catalog/api/serializers/media_serializers.py:281]: Hey @sarayourfriend, thank you for the review will update the code once we finalise the help text.

File: api/catalog/api/serializers/media_serializers.py
Comment (AetherUnbound) [inline api/catalog/api/serializers/media_serializers.py:282]:
```suggestion
        help_text="The accuracy of a machine-generated tag. Human-generated "
        "tags have a null accuracy field.",
```

File: api/catalog/api/serializers/media_serializers.py
Comment (AetherUnbound) [reply api/catalog/api/serializers/media_serializers.py:281]: +1 to that, I've added a suggestion for that wording.

**Review by AetherUnbound — APPROVED** [review]
Thanks for the contribution @sahil-R! This looks good and can be merged as soon as we update the help text.

File: api/catalog/api/serializers/media_serializers.py
Comment (sahil-R) [reply api/catalog/api/serializers/media_serializers.py:281]: Hey @AetherUnbound, @sarayourfriend  and @dhruvkb, I have made the required changes, and the request has also passed all the mandatory tests. Thank you for your help and suggestions.

Comment (AetherUnbound) [reply api/catalog/api/serializers/media_serializers.py:281]: Excellent, thank you @sahil-R!

**Pull request merged**
